snackpub/skaffold.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: skaffold/v2beta29
+kind: Config
+metadata:
+  name: snackpub
+build:
+  platforms: ['linux/amd64']
+  artifacts:
+    - image: gcr.io/exponentjs/snackpub
+      sync:
+        manual:
+          - src: src/**/*
+            dest: /snack/app
+      docker:
```

This is all fine; you can also set the required build context in the skaffold yaml itself.

```suggestion
      context: ../
      docker:
```

---

have to revert the change in https://github.com/expo/snack/pull/377/commits/2728b09d0375a1f3e4d54b54323c2e6976001727. it will transfer context files out from snack root, e.g. `/path/to/snack` in the git repo and the `context: ../` will load the files from `/path/to` which is not quite correct.
will keep the code as-is.